WebRoot/js/zimbraAdmin/resource/model/ZaSignature.js

```diff
@@ -103,6 +103,9 @@ function(by, val) {
 			&& !ZaZimbraAdmin.haveTargetRight(ZaItem.RESOURCE,ZaResource.VIEW_RESOURCE_MAIL_RIGHT,ZaAccount.getDomain(this.name))) {
 		return;
 	}
+	if(!ZaZimbraAdmin.isGlobalAdmin() && this.attrs[ZaResource.A_accountStatus] != ZaResource.ACCOUNT_STATUS_ACTIVE) {
```

Comment:
are we going to send request even if account is not active but logged in user is global admin?

Reply:
GetSignaturesRequest throws an exception only when a delegated admin can manage a signature of a calendar resource of a domain and the status of a calresource is Closed.
So we don't need to prevent GetSignaturesRequest from being sent if logged in user is global admin.

[Global admin]
- If the status of calresource is Active, an exception is not thrown. A server returns signature data.
- If the status of calresource is Closed, an exception is not thrown. A server returns signature data.

[Delegated admin]
- If the status of calresource is Active, an exception is not thrown. A server returns signature data.
- If the status of calresource is Closed, an exception is thrown.